Kickstarter-iOS/Views/Cells/FindFriendsCell.swift

```diff
@@ -0,0 +1,70 @@
+import Foundation
+import KsApi
+import Library
+import Prelude
+
+final class FindFriendsCell: UITableViewCell, ValueCell, NibLoading {
+  @IBOutlet fileprivate weak var arrowImageView: UIImageView!
+  @IBOutlet fileprivate weak var descriptionLabelContainer: UIView!
+  @IBOutlet fileprivate weak var disabledDescriptionLabel: UILabel!
+  @IBOutlet fileprivate weak var separatorView: UIView!
+  @IBOutlet fileprivate weak var titleLabel: UILabel!
+
+  private let viewModel: FindFriendsCellViewModelType = FindFriendsCellViewModel()
+
+  func configureWith(value: SettingsCellValue) {
+    let user = value.user
+    self.viewModel.inputs.configure(with: user)
+  }
+
+  override func bindViewModel() {
+    super.bindViewModel()
+
+    self.descriptionLabelContainer.rac.hidden = self.viewModel.outputs.disabledDescriptionLabelShouldHide
+
+    self.viewModel.outputs.isDisabled
+      .observeForUI()
+      .observeValues { [weak self] (isDisabled) in
+        self?.updateStyles(isDisabled: isDisabled)
+    }
+  }
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = titleLabel
+      |> settingsTitleLabelStyle
+      |> UILabel.lens.text .~ SettingsCellType.findFriends.title
+
+    _ = arrowImageView
+      |> settingsArrowViewStyle
+
+    _ = disabledDescriptionLabel
+      |> settingsDescriptionLabelStyle
+      |> UILabel.lens.text
+      .~ "Following must be on to find Facebook friends. Following can be changed in Account > Privacy."
```

Comment:
Does this need to be localized? I've found it useful to use the `localizedString()` function while waiting for translations so that it's easy to do a find/replace on that once the translations are ready. It's also safe to merge because `localizedString()` is what all of the `Strings` functions are using in any case.